client/layout/masterbar/logged-in.jsx

```diff
@@ -96,6 +96,7 @@ export default React.createClass( {
 					{ this.translate( 'New Post' ) }
 				</Publish>
 				<Item
+					tipTarget="me"
```

Comment:
Maybe something clearly explicitly relating it to the React infrastructure behind it?

Like: `tourStepID` and `data-tour-step-id`? (→ because `?tour` is the trigger).